src/components/task-plan/homework.cjsx

```diff
@@ -68,6 +68,7 @@ ChooseExercises = React.createClass
         hide={hide} />
 
       <PinnedHeaderFooterCard
+        containerBuffer={50}
```

Comment:
This is describing the extra space desired between the bottom of the pinned header and the start of the main content, [line 135 of the component](https://github.com/openstax/tutor-js/pull/369/files#diff-0ca5c713d2c99710c0c8b04c3da4e07bR135).  The bulk of the margin height is still dependent on the height of the header itself.  Since the height of the header will change based on state changes/header content or window resizing, I don't know if we can make the size explicit =(